### ISSUE TITLE
Post Notification Emails: "Listen on Spotify" links to the post URL, not Spotify

### ISSUE THREAD

**Issue by donalirl** [Automattic/wp-calypso#92456]
### Quick summary

New post notification emails contain a "Listen on Spotify" button when the post contains a Spotify embed, but the button's link is wrong.

### Steps to reproduce

1. Create a new post
2. In the body of a post, embed a Spotify link such as https://open.spotify.com/track/3C6zk9mkmv8f522AghQD9G?si=f755e920481840b0
3. Publish the post
4. Check the new post notification email sent to subscribers
5. Click on "Listen on Spotify"

![Screenshot 2024-07-07 at 12 10 46 PM](https://github.com/Automattic/wp-calypso/assets/30727666/a4f2d220-3b3b-470a-aa32-cc7738330881)

### What you expected to happen

I expected to be taken to spotify.com to listen to the full track.

### What actually happened

The link takes the user to the post URL (where just a preview of the Spotify track is shown)

### Impact

All

### Available workarounds?

No but the platform is still usable

**Comment by donalirl** [comment]
This might have been a recent change, because as recently as July 2 there was no "Listen on Spotify" button. Instead, the full Spotify URL displayed instead as per #8442801-zen

**Comment by simison** [comment]
We just implemented these audio embeds in the emails with Zap. 👋 

We debated if to link to Spotify or to the post, and I wanted us to link to the post since that's where they can listen as well, have commenting available, have potentially transcript/accompanying text available, and aren't redirected outside the blog.

Similar case are YouTube video embeds.

We did leave the copy somewhat confusing though. @crisbusquets @enejb thoughts?

**Comment by crisbusquets** [comment]
> We did leave the copy somewhat confusing though. @crisbusquets @enejb thoughts?

I agree the copy is confusing. Can we use something more generic? In Figma I had `Listen to the audio in the post`.

**Comment by simison** [comment]
That copy sounds good to me 👍

**Comment by enejb** [comment]
@simison and @crisbusquets I am happy to change the copy from "Listen on Spotify" to "Listen to the audio in the post" 
but as I understand it the user is not confused by wording but by where the link takes once they click on it. 

Another problem that currently exits if we link users to just the post is that if there are multiple audio blocks that they end they have to find the audio and click on it. There are already a number of ways for the user to get to the post and experience the audio clips there so using the the audio buttons to do the same things here is not really solving the issue. IMHO I think we should link to the different sources. And provide folks with a heads up with the wording such as "Listen on Spotify" that they will be take to Spotify or a different service that the URL might take them to. This will give the readers the choice how they want to experience the audio piece.

**Comment by donalirl** [comment]
> Listen to the audio in the post

Something to consider here is that the Spotify embed in the post will only play a short preview of the audio (I think 30 seconds), not the full audio. The user then needs to visit Spotify to hear the audio in full.

So if the user's goal is to listen to the audio, they'll have extra clicks with the current flow: _email → post → spotify_ If the button says "Listen to the audio in the post", it could be seen as misleading since they can't listen to the _full_ audio in the post.

If you decide to keep the button linking to the post, then I feel button copy like "View post" will be the clearest.

Or if you decide to link the button to the Spotify URL, then the current "Listen on Spotify" copy will be the clearest.

**Comment by crisbusquets** [comment]
> So if the user's goal is to listen to the audio, they'll have extra clicks with the current flow: _email → post → spotify_ If the button says "Listen to the audio in the post", it could be seen as misleading since they can't listen to the _full_ audio in the post.

Thanks for the input, @donalirl. This is aligned with the approach @enejb shared above. It works for me, as long as the copy is clear and people know where they're going to land.

**Comment by simison** [comment]
Yeah,  came from disconnect between the copy & action — and thanks for noting the "preview" nature of spotify embeds, makes sense now. 👍

**Comment by simison** [comment]
> This will give the readers the choice how they want to experience the audio piece.

Right, I was thinking more about the publisher who decided to embed content on their blog rather than link elsewhere, but it really depends on nature of embedded service like seen with this Spotify example.

**Comment by crisbusquets** [comment]
I'd say we do what @donalirl suggested (for example, if it's a Spotify embed, open Spotify) and keep track of tickets to see if site/newsletter owners have something to say. WDYT?

**Comment by enejb** [comment]
A fix for this was shipped! cc @donalirl in case you want to let the user know.
